projects/blue-move-mart/index.js

```diff
@@ -1,12 +1,14 @@
 const http =  require('../helper/http')
-const bluemoveApi =  'https://aptos-mainnet-api.bluemove.net/api/market-info/tvl';
+// const bluemoveApi =  'https://aptos-mainnet-api.bluemove.net/api/market-info/tvl';
+const bluemoveApi  = 'https://aptos-mainnet-api.bluemove.net/api/market-info/stake-tvl';
```

Comment:
I am getting request forbidden for this, is there any way to get this on chain? 
also, can you pull the latest code and count this under `staking` instead?